DynamicGapPlanner:
    min_resoln: 1
    goal_tolerance: 0.5
    yaw_goal_tolerance: 3.15 
    xy_goal_tolerance: .1

    epsilon2: 0.20 # 0.18 0.09
    epsilon1: 0.20 # 0.18 0.09
    r_inscr:  0.20

    ctrl_ahead_pose: 2
    reduction_threshold: 3.1415926
    max_idx_diff: 256
    k_fb_x: 2
    k_fb_y: 2
    k_fb_theta: 0.8

    projection_inflated: False
    fig_gen: True
    k_po_x: 1
    k_po_theta: 3
    k_CBF: 1.0

    r_min: 0.35
    r_norm: 1.0
    r_norm_offset: 0

    synthesized_frame: True
    scale: 1.0
    integrate_maxt: 5
    integrate_stept: 0.50

    num_feasi_check: 30
    projection_operator: False
    holonomic: True

    debug_viz: True
    max_pose_pen_dist: 0.5
    inf_ratio: 1.25
